package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "screwdriver-data-schema",
-  "version": "12.1.0",
+  "version": "13.0.0",
```

Comment:
If we remove the key change, this is not-backwards incompatible and we can set this back to `12.1.0`